sccm/sum/deploy-use/third-party-software-updates.md

```diff
@@ -31,18 +31,18 @@ Beginning with version 1806, the **Third-Party Software Update Catalogs** node i
 
 ## Additional requirements when the SUP is remote from the top-level site server 
 
-1. SSL must be enabled on the SUP when it's remote. 
+1. SSL must be enabled on the SUP when it's remote. This requires a server authentication certificate generated from an internal certififcate authority or via a public provider.
     - [Configure SSL on WSUS](https://docs.microsoft.com/windows-server/administration/windows-server-update-services/deploy/2-configure-wsus#bkmk_2.5.ConfigSSL)
         - When you configure SSL on WSUS, note some of the web services and the virtual directories are always HTTP and not HTTPS. 
         - Configuration Manager downloads third-party content for software update packages from your WSUS content directory over HTTP.   
     - [Configure SSL on the SUP](../get-started/install-a-software-update-point.md#configure-ssl-communications-to-wsus)
 
-2. To allow the creation of the self-signed WSUS certificate: 
+2. When setting third-party updates WSUS signing certificate configuration to **Configuration Manager manages the updates** in the Software Update Point Component Properties, the following is required to allow the creation of the self-signed WSUS certificate: 
```

Comment:
Content looks good. A few editorial comments:
- "When setting *the* third-party updates..." (add "the", italics added for emphasis here, not needed in final)
- "...the following configurations are required to allow..." (add a noun after "following" to help the reader understand what you're referring to)